### ISSUE TITLE
NULL is converted to True

### ISSUE THREAD

**Issue by shilobar** [MattFanto/orjson-ddb#5]
Hi!
The "NULL" type is not being converted correctly:

```
import json

from boto3.dynamodb.types import TypeDeserializer
from orjson_ddb import orjson_ddb

record = b'{"NULL":true}'
assert orjson_ddb.loads(record) == TypeDeserializer().deserialize(json.loads(record))
# True != None
```

**Comment by mpuhacz** [comment]
I also encountered the same problem, @MattFanto do you think it's possible to get a fix for that? Happy to help with this

**Comment by MattFanto** [comment]
Ah sorry I missed the notification for the issue, yeah shouldn't be difficult to fix in theory it would just need a special case here for "NULL" type https://github.com/MattFanto/orjson-ddb/blob/8ce73e7da6c1781da525a97652b3d5f1415a87a5/src/deserialize/yyjson.rs#L254 
I'm not sure when I will be able to work on it but if you want open a PR @mpuhacz it would super appreciated

**Comment by MattFanto** [comment]
Solved in 0.2.1

**Comment by mpuhacz** [comment]
@MattFanto thank you for the fix!
